### ISSUE TITLE
Remove ClientContextFactory use from mail examples.

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#8603]
|[<img alt="Lukasa's avatar" src="https://avatars.githubusercontent.com/u/1382556?s=50" width="50" height="50">](https://github.com/Lukasa)| @Lukasa reported|
|-|-|
|Trac ID|trac#8603|
|Type|enhancement|
|Created|2016-07-15 09:01:50Z|
|Branch|https://github.com/twisted/twisted/tree/8603-lukasa-clientcontextfactory|

This got missed in [#8595](https://github.com/twisted/twisted/issues/8595): imap4client still has a ClientContextFactory in it.

<details><summary>Searchable metadata</summary>

```
trac-id__8603 8603
type__enhancement enhancement
reporter__Lukasa Lukasa
priority__normal normal
milestone__None None
branch__8603_lukasa_clientcontextfactory 8603-lukasa-clientcontextfactory
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__None None
time__1468573310046389 1468573310046389
changetime__1468781792429786 1468781792429786
version__None None
owner__Lukasa Lukasa

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="Lukasa's avatar" src="https://avatars.githubusercontent.com/u/1382556?s=50" width="50" height="50">](https://github.com/Lukasa)<a name="note_1"></a>|@Lukasa commented|
|-|-|

In changeset 406468b6851149ac4a415d1c3526047168e3675b
```
#!CommitTicketReference repository="" revision="406468b6851149ac4a415d1c3526047168e3675b"
Add topfile for #8603.
```

**Comment by twisted-trac** [comment]
|[<img alt="adiroiban's avatar" src="https://avatars.githubusercontent.com/u/204609?s=50" width="50" height="50">](https://github.com/adiroiban)<a name="note_3"></a>|@adiroiban set owner to @Lukasa|
|-|-|

Thanks. Please merge!

**Comment by twisted-trac** [comment]
|<img alt="GitHub's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50"><a name="note_4"></a>|GitHub set status to `closed`|
|-|-|

In changeset 02207ce4cedb742ae59994a1faef5294b2cd6468
```
#!CommitTicketReference repository="" revision="02207ce4cedb742ae59994a1faef5294b2cd6468"
Merge 8603-lukasa-clientcontextfactory: Remove explicit ContextFactory from mail example.

Author: lukasa
Reviewer: rodrigc
Fixes: #8603
```
